### ISSUE TITLE
Which branch is scanned by default when scanning git repos?

### ISSUE THREAD

**Issue by umangkedia** [olacabs/jackhammer#39]
Hi,

Which branch is scanned by default when scanning git repos? Can I change the branch somewhere?

**Comment by kmadhusudhan** [comment]
by default it scans **Master** branch . you can change  branch name while adding target 

<img width="1148" alt="screen shot 2017-06-12 at 11 51 17 am" src="https://user-images.githubusercontent.com/5602017/27021386-917309a4-4f65-11e7-8491-aba5e5d01809.png">

**Comment by umangkedia** [comment]
@KMadhuSudhan Hey, Can I change it in some config if possible? Also, why does this option doesn't come in corporate scan? By default, I want to scan develop branch always.

**Comment by kmadhusudhan** [comment]
We removed this feature due to the multi selection of teams and repos.Changing the branch name  available only in personal scans .

**Comment by umangkedia** [comment]
@KMadhuSudhan Makes sense. Can you point me to file where I can change the branch from master to develop and rebuild for internal use?

**Comment by kmadhusudhan** [comment]
you can change in  web/app/app/workers/base.rb . 
line no : 41    
**code:** branch = branch.present? ? branch : 'master'
**change to:**  branch = branch.present? ? branch : 'develop'
